src/renderer/components/Platform/AppDetails.js

```diff
@@ -87,7 +84,10 @@ type Props = {
 
 const AppDetails = ({ manifest }: Props) => {
   const { t } = useTranslation();
-  const language = useSelector(languageSelector);
+  const description = t(
+    `manager.apps.content.${manifest.id}.shortDescription`,
```

Comment:
Small thing, most of these use quotes `"` not back ticks ` ` `

Reply:
backtick is necessary for the templating of ${manifest.id} here, so seems legit.

Reply:
Backticks are used in Javascript to inject variables to a string, that's why I use them here

Reply:
https://media.giphy.com/media/jUwpNzg9IcyrK/giphy.gif

Reply:
Sorry, I suppose that I forgot to refresh the page before sending my message, didn't see that gre already responsed.